src/NServiceBus.Core.Tests/Sagas/ComponentTests/SimpleSagaEntitySaga.cs

```diff
@@ -0,0 +1,23 @@
+namespace NServiceBus.Persistence.ComponentTests
+{
+    using System;
+    using System.Threading.Tasks;
+
+    public class SimpleSagaEntitySaga : Saga<SimpleSagaEntity>, IAmStartedByMessages<StartMessage>
+    {
+        public Task Handle(StartMessage message, IMessageHandlerContext context)
+        {
+            throw new NotImplementedException();
+        }
+
+        protected override void ConfigureHowToFindSaga(SagaPropertyMapper<SimpleSagaEntity> mapper)
+        {
+            mapper.ConfigureMapping<StartMessage>(msg => msg.SomeId).ToSaga(saga => saga.OrderSource);
+        }
+    }
+
+    public class SimpleSagaEntity : ContainSagaData
+    {
+        public string OrderSource { get; set; }
```

Comment:
not introduced in this PR https://github.com/Particular/NServiceBus/blob/develop/src/NServiceBus.Core.Tests/Persistence/InMemory/SimpleSagaEntitySaga.cs#L21   so should be raised on a diff issue